backend/infrahub/graphql/mutations/main.py

```diff
@@ -101,15 +100,17 @@ async def mutate(cls, root: dict, info: GraphQLResolveInfo, *args: Any, **kwargs
 
             data = await obj.to_graphql(db=context.db, filter_sensitive=True)
 
-            message = messages.EventNodeMutated(
+            event = NodeMutatedEvent(
                 branch=context.branch.name,
                 kind=obj._schema.kind,
+                namespace=obj._schema.namespace,
```

Comment:
It might have a bit limiting previously to only have `kind` but I think the addition of namespace here gets a bit weird if we keep kind. We should either just leave it as "kind" or remove kind if we have "name" and "namespace.

Reply:
I removed it for now, we can revisit that later but having just the kind is not enough in my perspective if we need to do some filtering